src/main/scala/org/hammerlab/guacamole/likelihood/Likelihood.scala

```diff
@@ -157,7 +157,7 @@ object Likelihood {
             case (allele, alleleFraction) =>
               alleleElementProbabilities(alleleToIndex(allele), ::) * alleleFraction
           }
-          sum( log( sum(alleleRows) )) + math.log(prior(genotype)) * depth
+          sum( log( sum(alleleRows) + 1e-10 )) + math.log(prior(genotype))
```

Comment:
At the very least I think we should put together a test case for the `sum(alleleRows) == 0` scenario, lmk if you have ideas about how to arrange that.